README.md

```diff
@@ -24,6 +24,11 @@ source .travis/s2n_setup_env.sh
 .travis/s2n_travis_build.sh
 ```
 
+### Have a Question?
+If you have any questions about Submitting PR's, Opening Issues, s2n API usage, or something similar, we have a public chatroom available here to answer your questions: https://gitter.im/awslabs/s2n
+
+Otherwise, if you think you might have found a security impacting issue, please instead follow: https://aws.amazon.com/security/vulnerability-reporting/
```

Comment:
This link is already posted in the `Security issue notifications` section of this README. Perhaps a markdown anchor link to that seciton would be better?